src/common/options.cc

```diff
@@ -2349,6 +2349,12 @@ std::vector<Option> get_global_options() {
     .set_default(10.0)
     .set_description("Seconds before in-flight op is considered 'laggy' and we query mon for the latest OSDMap"),
 
+    Option("objecter_homeless_timeout", Option::TYPE_FLOAT, Option::LEVEL_ADVANCED)
+    .set_default(0)
+    .set_description("Timeout homeless session : Duration (in fractional seconds) after which the op is cancelled")
+    .set_long_description("It will cancel the op associated with a homeless session after the specified duration."
+                          "Homeless session means that there is no availiable osd in the PG to process the I/O"),
```

Comment:
need a space between `specified duration."` and `"Homeless session`

Reply:
Done